<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>666</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>161</width>
         <height>0</height>
        </size>
       </property>
       <widget class="QLabel" name="label_size">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>170</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>容量：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>210</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>JEDEC编码：</string>
        </property>
       </widget>
       <widget class="QComboBox" name="combo_vendor">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>90</y>
          <width>111</width>
          <height>28</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Winbond</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MXIC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GigaDevice</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ATMEL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>未知</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="combo_model">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>130</y>
          <width>111</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>厂商：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>130</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>型号：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>类型：</string>
        </property>
       </widget>
       <widget class="QComboBox" name="combo_memType">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>111</width>
          <height>28</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>25 SPI Flash</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24 EEPROM</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="btn_detect">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>330</y>
          <width>81</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>检测芯片</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_open">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>290</y>
          <width>71</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>打开</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_save">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>330</y>
          <width>71</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
       <widget class="QComboBox" name="combo_ports">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>111</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>端口：</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_flush">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>290</y>
          <width>81</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>刷新端口</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_code">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>240</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_cropping">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>370</y>
          <width>71</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>裁剪</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="HexView" name="hexView" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_erase">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>擦除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_checkEmpty">
           <property name="text">
            <string>查空</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_read">
           <property name="text">
            <string>读取</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_write">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>写入</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_check">
           <property name="text">
            <string>校验</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>420</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_cropping"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_L">
    <property name="title">
     <string>语言(&amp;L)</string>
    </property>
    <addaction name="action_select_zh_CN"/>
    <addaction name="action_select_en"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_L"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+S</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Q</string>
   </property>
  </action>
  <action name="action_select_zh_CN">
   <property name="text">
    <string notr="true">简体中文</string>
   </property>
   <property name="iconText">
    <string notr="true">简体中文</string>
   </property>
   <property name="toolTip">
    <string notr="true">简体中文</string>
   </property>
  </action>
  <action name="action_select_en">
   <property name="text">
    <string notr="true">English</string>
   </property>
   <property name="iconText">
    <string notr="true">English</string>
   </property>
   <property name="toolTip">
    <string notr="true">English</string>
   </property>
  </action>
  <action name="action_cropping">
   <property name="text">
    <string>裁剪</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HexView</class>
   <extends>QWidget</extends>
   <header>hexview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
